src/explore-education-statistics-frontend/src/modules/download/DownloadIndexPage.tsx

```diff
@@ -57,7 +58,16 @@ const DownloadIndexPage: NextPage<Props> = ({ themes = [] }) => {
       </div>
 
       {themes.length > 0 ? (
-        <Accordion id="downloads">
+        <Accordion
+          id="downloads"
+          onToggle={accordionSection => {
+            logEvent(
+              'Accordion',
```

Comment:
Shouldn't these be categorised by page rather than into one big 'Accordion' category? Seems like it wouldn't be very useful to categorise it like this.

Same applies for the other Accordions too.